src/animation/gravity.ts

```diff
@@ -86,32 +81,28 @@ export const gravity = (params: GravityParams): [() => void] => {
     if (steps > 0) {
       // Apply the gravitational force once for each step.
       for (let i = 0; i < steps; i++) {
-        state.mover = applyForceForStep(
-          state,
-          params.config.threshold || DEFAULT_THRESHOLD
-        );
+        state.mover = applyForceForStep(state);
       }
     } else {
-      state.mover = applyForceForStep(
-        state,
-        params.config.threshold || DEFAULT_THRESHOLD
-      );
+      state.mover = applyForceForStep(state);
     }
 
     /**
-     * Conditions for stopping the animation. If the mover is within a specified
-     * threshold of the attractor, the gravitational force will become so strong
-     * that its acceleration will increase by orders of magnitude each frame. Similarly,
-     * a mover may still get catapulted by the gravitational
+     * Conditions for stopping the physics animation. For single animations with
+     * a discrete from / to pair, we want to stop the animation once the mover has
+     * reached the attractor. We can know its done by checking that the mover has
+     * not overshot the attractor.
```

Comment:
Super nit, am I mis-reading the last sentence of this comment?  Is the "not" in "has not overshot" correct?  Isn't it supposed to be, it's done when it "has overshot the attractor"?

Reply:
Yep, just pushed a commit to fix that 😅